### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by akinmolaomotlani**
1. Added html and css files to the project 
2. installed the html,css linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -319,5 +319,66 @@ <h3 class="about-heading-primary">About Myself</h3>
...
+
+          </div>
+
+        </form>
+
+      </div>
+
+    </div>
+
+    <button class="btn send" type="button">Send</button>
```
Comment (Abenezer-Tilahun) [inline index.html:380]: - [ ] Kindly I recommend you to include the button inside the form to submit the form successfully and as stated in the project requirement Kindly Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @akinmolaomotlani @tajulafreen   , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._

- [ ]  As stated in the project requirement Kindly make sure that each card on the board has the **number of points in the [Xpt**] prefix. That number should indicate the number of points that show how complex the task is. 1 point = 1 hour of work.
a description of the task and **Has a description of the task.**

Capture

Capture

- [ ] But in the case of yours Kanban board there is no he **number of points in the [Xpt**] prefix on all of your cards and some of your cards has no  **a description of the task.** Just Like the following cards, So Kindly Fix it before resubmit for review.

Capture

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -319,5 +319,62 @@ <h3 class="about-heading-primary">About Myself</h3>
...
+          </p>
+        </div>
+
+        <div class="cta-form-box">
+          <form
+            action="https://formspree.io/f/{form_id}"
+            method="post"
+            class="cta-form"
+          >
+            <div>
```
Comment (Shedrack-Sunday) [inline index.html:339]: - [ ] As highlighted by the previous reviewer, kindly replace the `{form_id}` with the form in your [formspree account](https://formspree.io/html/) so your form can be successfully submitted. The link highlighted has a detailed approach to that.
- I'll advice you make sure it submits before sending for another review 👍

File: index.html [hunk 3]
```diff
@@ -319,5 +319,62 @@ <h3 class="about-heading-primary">About Myself</h3>
...
+              <label for="name"></label>
+              <input
+                id="full-name"
+                type="text"
+                placeholder="full name"
+                name="name"
+                required
```
Comment (Shedrack-Sunday) [inline index.html:346]: - [ ] As rightly suggested by the last reviewer, kindly add validations on this name input field a length of `30` as specified on the project requirements. 
- This makes sure the input from the user is very reasonable 👍 

## Requirements:
image

File: style.css [hunk 4]
```diff
@@ -364,3 +364,105 @@ li {
...
+  padding-top: 180px;
+  font-family: "poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 56px;
+  letter-spacing: 0.37px;
+  color: #fff;
+  margin-bottom: 12px;
+  text-align: center;
```
Comment (Shedrack-Sunday) [inline style.css:397]: - [ ] Nice job using Flexbox as rightly stated by the last reviewer 👏 . However, you still need to adjust the startin point of the input in the text area in the form. From the comparison below, you can see they are clearly not the same as given in the Figma design.
- In the `button`

## Figma:
image

## Your design:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @akinmolaomotlani       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use implementation of  a couple of highlighted changes ✔️    

  
# Required Changes  🌵 

- [x] You made some impressive efforts as regards the previous review suggestion of  which I must say is a job well done. However, there's one you missed 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: [OPTIONAL]
- [ ] please i kindly suggest that you apply the appropriate border-radius your design for each input and textareas as shown in the figma template below. It is important that your design matches closely the figma template as it shows how much attention to details you pay as a developer :+1:  :sparkles: 
- [ ] Also work on improving the fonts of the placeholder text as well as theit color see the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=kCqJowj7ub2IRXn2-0) for this. :+1: 
- figma design
f-f
- your design
f-s

**Review by Efiamotu-1 — APPROVED** [review]
Hi @akinmolaomotlani , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**